project-icp/bee-pollinator-app
# Tw/blackbox feedback

## Overview

Two small spelling/capitalization changes to the April survey UI:

 - Capitalize November in the list of choices
 - Spell inadequate correctly

Created migration to update the database entries to spell INADEQUATE correctly.

Connects #570 

### Demo

<img width="589" alt="Screen Shot 2020-02-28 at 2 58 21 PM" src="https://user-images.githubusercontent.com/21046714/75583002-c81a6f00-5a3a-11ea-9143-1bea8ed0baea.png">

## Testing Instructions

* Ensure you have an April Survey in your database with 'INADEQUETE_FOOD_STORES'.
* Check out this branch
* Run `./scripts/manage.sh migrate`
* Your April Survey should now say ' INADEQUATE_FOOD_STORES'.
* Run `./scripts/beekeepers.sh start`
* Navigate to an April survey
* You should see 'November' and 'inadequate'
